src/app/handlers/flightsql.rs

```diff
@@ -73,7 +73,7 @@ pub fn normal_mode_handler(app: &mut App, key: KeyEvent) {
             let execution = Arc::clone(&app.execution);
             let _event_tx = app.app_event_tx.clone();
             tokio::spawn(async move {
-                let client = &execution.flightsql_client;
+                let client = execution.flightsql_client();
```

Comment:
I just made the fields non pub and made accessors so the API surface area was clearer

Reply:
Yup makes sense, i had started to use that approach for more recent work but i know theres some remaining pub stuff